app/assets/stylesheets/application.scss

```diff
@@ -417,6 +430,18 @@ footer ul li a:hover, footer ul li a:focus, footer ul li a:active, footer ul li
   text-decoration: none;
 }
 
+.checkbox-space{
+  margin: 0px 1px 0px 1px;;
+}
+
+.center-search{
```

Comment:
This little HTML secret FEers dont want you to know

Reply:
Tell us all their secrets...